playbook/app/pb_kits/playbook/pb_popover/docs/_popover_list.html.erb

```diff
@@ -22,14 +22,14 @@ const button = document.querySelector("#list")
 let buttonClicked = false
 
 const arrowDiv = document.querySelector("#arrow-icon")
-arrowDiv.innerHTML = '<i class="far fa-angle-down"></i>'
+arrowDiv.innerHTML = '<i> pb_rails("icon", props: { icon: "arrow-down", fixed_width: true }) </i>'
```

Comment:
Same. We shouldn't be wrapping the pb_rails with an <i>. 
<img width="546" alt="Screenshot 2024-09-25 at 10 56 54 AM" src="https://github.com/user-attachments/assets/80fab08b-5449-4c0f-adbd-b3d11f162523">

It looks like you are trying to inject the rails kit into the div, but it's never getting there. ^